ml-notebook/packages.txt

```diff
@@ -383,8 +390,9 @@ netcdf-fortran==4.5.4
 netcdf4==1.5.8
 nettle==3.6
 networkx==2.8
+ninja==1.10.2
 noise==1.2.2
-nomkl==1.0
+nomkl==2.0
```

Comment:
odd to see both 'mkl' and 'nomkl'

Reply:
nomkl is coming from https://github.com/pangeo-data/pangeo-docker-images/blob/b5c38d09ca44e66dfda529fbfeb15ba4789a3805/pangeo-notebook/environment.yml#L58, presumably mkl is coming with pytorch...